packages/manager/apps/dedicated/client/app/cdn/dedicated/manage/cdn-dedicated-manage.routes.js

```diff
@@ -0,0 +1,13 @@
+angular.module('App').config(($stateProvider) => {
+  $stateProvider.state('app.networks.cdn.dedicated.manage', {
+    redirectTo: 'app.networks.cdn.dedicated.manage.statistics',
+    translations: { value: ['.'], format: 'json' },
+    views: {
+      cdnMainView: {
+        templateUrl: 'cdn/dedicated/manage/cdn-dedicated-manage.html',
```

Comment:
Could be imported then (tell webpack to import it)?

Reply:
Could be, but it's not the point of this PR 🙂

Reply:
Done